pkgs/populate/default.nix

```diff
@@ -157,6 +157,7 @@ let
     fi
     ${rsync}/bin/rsync \
         ${optionalString (config.useChecksum or false) /* sh */ "--checksum"} \
+        ${optionalString target.sudo /* sh */ "--rsync-path=\"sudo rsync\""} \
```

Comment:
Using single quotes around `sudo rsync` would look slightly nicer. But it's OK to leave it like this.